src/validation.cpp

```diff
@@ -428,7 +428,7 @@ bool AcceptToMemoryPoolWorker(CTxMemPool& pool, CValidationState& state, const C
 
     // is it already in the memory pool?
     if (pool.exists(hash))
-        return state.Invalid(false, REJECT_ALREADY_KNOWN, "txn-already-in-mempool");
+        return state.Invalid(false, REJECT_DUPLICATE, "txn-already-in-mempool");
 
```

Comment:
I would have added braces on this and the other lines that are being changed to update the style.